java/ql/test/library-tests/frameworks/apache-commons-lang3/Test.java

```diff
@@ -192,8 +192,8 @@ void test() throws Exception {
         sink(StringUtils.rotate(taint(), 0)); // $hasTaintFlow
         sink(StringUtils.split(taint())); // $hasTaintFlow
         sink(StringUtils.split(taint(), ' ')); // $hasTaintFlow
-        sink(StringUtils.split(taint(), " ,; // $hasTaintFlow")); // $hasTaintFlow
-        sink(StringUtils.split(taint(), " ,; // $hasTaintFlow", 0)); // $hasTaintFlow
+        sink(StringUtils.split(taint(), " ,;")); // $hasTaintFlow
+        sink(StringUtils.split(taint(), " ,;", 0)); // $hasTaintFlow
```

Comment:
It doesn't really matter much, but I'd say that was intended to be just a space:

```suggestion
        sink(StringUtils.split(taint(), " ")); // $hasTaintFlow
        sink(StringUtils.split(taint(), " ", 0)); // $hasTaintFlow
```

Reply:
It isn't actually; these functions take a string which is a list of separator chars, so this is a realistic (albeit irrelevant) input